src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.3.22
+++++++
+* Removing `--fqdn` flag on az openshift commands
```

Comment:
This is a breaking change and cannot be done with a simple patch version. This needs to call out the breaking change and be bumped to version 2.4.0